### PR TITLE
17th Small Improvments

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- **God tweaks:**
  - The **Arcane**, **Rationalistic**, **Wealthy** religions have **the Ancestors** as their high gods.
  - The **Necromantic** religions have **the Death** as their high god, the **Druidic** religions have **the Death** as their evil god.
- The high elves' religion renamed **Belore**, the sun in Thalassian.
- Qiraji, Nerubian, Scourge provinces get big minus to Land Organisation, Garrison Size, but a big bonus to Levy Size,  Levy Reinforcement Rate and Morale of Armies.
- Centaur, Gnoll, Wolvar, Harpy, Hozen, Kobold, Murloc, Gorloc, Quilboar provinces get minus to Morale of Armies, Garrison Size, but a bonus to Levy Size and Levy Reinforcement Rate.
- You can't **Change Childhood Focus to Heritage** and **Ask Permission for Heritage Focus** by right-clicking children with the guardian of the foreign culture if the switch to this culture isn't allowed. For example, a murloc child (**Murloc** culture, **Nerglish** religion) shouldn't allow the decisions if the guardian is a naga.
- Removed +3 and +4 health bonuses from many immortal races because it stopped the game from pruning these characters. Other races are limited to the +1 health bonus max.
- Being a historical character or immune to pruning aren't reasons to block characters from being expelled via the **Ask to Leave Court**.

## Script changelog (ignore if you can't read CK2 script):
- Cleared `ongoing_fury_of_wolf_chain_flag` when `worgens_in_azeroth_flag` is set.
- Included `can_switch_culture_character_trigger` in `can_switch_culture_to_prev_character_trigger`.
- Used `can_be_asked_to_leave_court_by_root_trigger` in `LT.61008`, a trigger is similar to `can_be_asked_to_leave_court_by_from_trigger` used in `ask_to_leave_court`.
- Added `can_be_asked_to_leave_court_by_root_trigger` and `can_be_asked_to_leave_court_by_from_trigger`.

# How to test:
It should be easy to test, but if you have questions, feel free to ask them in the comments.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2981,6 +2980,8 @@ EVTDESC_WCTFT_600;Today [From.GetTitledFirstNameWithNick] announced that [From.G
 bloodline_effect_scarlet_king;Can �GVassalize�! the �YScarlet Crusade�!;;;;;;;;;;;;;x
 vassalize_d_scarlet_crusade;Vassalize the Scarlet Crusade;;;;;;;;;;;;;x
 vassalize_d_scarlet_crusade_desc;As a long-standing ally of the Scarlet Crusade, they could be convinced to become vassals of the realm if they see us fit.;;;;;;;;;;;;;x
+swarm_culture_modifier;Swarm Culture Modifier;;;;;;;;;;;;;x
+numerous_cowards_culture_modifier;Numerous Cowards Culture Modifier;;;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_zuwc_united_localisation.csv:2984]: > Numerous Cowards Culture

Imo, that sounds really bad. Idk what to suggest, but I would like to use something more neutral.

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv:2984]: > > Numerous Cowards Culture
> 
> Imo, that sounds really bad. Idk what to suggest, but I would like to use something more neutral.

Quantity Culture Modifier?

Comment (MemeWizard) [reply localisation/00_zuwc_united_localisation.csv:2984]: Sounds better

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Done in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1424/commits/8a155e5992521f1ff570d59b83a571ae90786d82.

File: common/religions/secular_group.txt [hunk 2]
```diff
@@ -214,12 +214,12 @@ secular_group = {
...
-		high_god_name = LOKEN
+		high_god_name = GOD_LOKEN
 		god_names = {
-			GOD_YOGGSARON THE_SHADOW
+			GOD_LOKEN
 		}
 		evil_god_names = {
-			THE_LIGHT THE_ORDER
+			GOD_YOGGSARON
 		}
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply common/religions/secular_group.txt:223]: > Are you sure about this change?

Yes, Loken went mad at the very end.
https://wow.gamepedia.com/Loken

**Comment by KoiKarp** [comment]
after about 2 weeks and a half from the start of the game, Medivh always converts to Sargerite unless you play him

**Comment by KoiKarp** [comment]
> after about 2 weeks and a half from the start of the game, Medivh always converts to Sargerite unless you play him

Same with Illidan, he doesn't convert religion but get's released from prison almost instantly after the start of the game

**Comment by KoiKarp** [comment]
Cairne starts with his Runespear unequiped

**Comment by zumbak04** [comment]
@KoiKarp reee, thats all not related to the PR. Give a real review!

**Pull request merged**